src/lib.rs

```diff
@@ -121,29 +126,36 @@ pub fn send(channel: &UnixStream, wrapped: FdImplementor) -> Result<(), String>
 
     iov.iov_base = &mut fdtype as *mut i32 as *mut c_void;
     iov.iov_len = size_of_val(&fdtype);
-    
+
     message.msg_control = controlbuf.as_mut_ptr() as *mut c_void;
-    message.msg_controllen = size_of_val(&controlbuf);
+    message.msg_controllen = size_of_val(&controlbuf) as c_uint;
```

Comment:
this `as c_uint` breaks the Linux compilation it seems.
We may risk to have to duplicate code here :/